### DISCUSSION TITLE
Database URL: Not able to connect to the database even after encoding the password that contained special characters?

### DISCUSSION THREAD

**Discussion by muhammadfaisal1194** [prisma/prisma#23049] [Q&A]
### Bug description

I'm trying to connect the AWS RDS to Prisma, 1stly it's working fine when I connect to pgAdmin using the same credentials. But when I tried Prisma, I came to know that my password should be encoded as it contained some special characters. After encoding my password length is 142 characters and it's still not connecting and throwing the same error.
"Authentication failed against database server".
After carefully going through the docs and issues on Github, I'm just suspecting that there might be an issue with my connection string length as the password is too long, other than there seems no issue,

### How to reproduce

1. Just use a long password that contains more than 140 characters.

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @muhammadfaisal1194 👋 

Thank you for raising this question.

Is it possible to send a disposable URL that is percent encoded and of the same length you described so I can try to reproduce it from my end and confirm this behaviour?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
